x-pack/plugins/spaces/server/routes/api/internal/get_active_space.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { wrapError } from '../../../lib/errors';
+import { InternalRouteDeps } from '.';
+import { createLicensedRouteHandler } from '../../lib';
+
+export function initGetActiveSpaceApi(deps: InternalRouteDeps) {
+  const { internalRouter, spacesService } = deps;
+
+  internalRouter.get(
+    {
+      path: '/internal/spaces/_active_space',
```

Comment:
__question:__ do we have some convention regarding `_` prefix in `_active_space`?

Reply:
I went back and forth for this specific route. We've been using the `_` prefix to denote routes that aren't strictly REST based. Most examples I've seen have been for "RPC" style calls, but I thought the `_` prefix would make it a little clearer that `active_space` wasn't a real space id, but rather a concept

Reply:
Thanks for explaining. So we kind of denote "operation" or "API sub domain" with `_` prefix, I see. That makes sense.

I'm not sure how I feel about differentiating between REST and non-REST though, difference is usually rather illusive and having something like `/api/spaces/_space/...`would probably make sense as well imo, but it's out of scope.

Reply:
Would you rather see `/internal/spaces/active_space` for this specific route? I'd be fine with either here

Reply:
I see we already have `/api/spaces/_copy_saved_objects` and `/api/spaces/_resolve_copy_saved_objects_errors`, `/internal/spaces/_active_space` would probably align better with those. So let's keep it as is and if we ever change our mind on `_` prefix we update it once and everywhere.